server/middlewares/verifyUser.js

```diff
@@ -0,0 +1,11 @@
+import jwt from 'jsonwebtoken';
+
+const verifyUser = (req, res, next) => {
```

Comment:
i think this function should be something like "checkToken" or "decodeToken"

Reply:
same here, I agree with @mosinmiloluwa01